components/button/button-split.js

```diff
@@ -78,36 +80,55 @@ class ButtonSplit extends FocusMixin(PropertyRequiredMixin(LocalizeCoreElement(L
 			::slotted(:not(d2l-button-split-item)) {
 				display: none;
 			}
+			.main-action-focus-visible .d2l-dropdown-opener {
```

Comment:
I tried using `:has(:focus-visible)` within `d2l-button-split`, but :`has()` only works within the same DOM scope. Hence the need for some JavaScript to detect focus-visible on the button buried inside of `d2l-button`.